### PR TITLE
FE-2159 - [Link Component] - remove console error

### PR DESCRIPTION
**Pull request by dawidzarzycki**
# Description
- refactored link component to be more automated 
- removed console error
- updated styling in related components

# Testing Instructions
- it should be good to tests every component that uses Link component to make it look the same as it is in the master branch

### CODE REVIEW

File: change_log/next/FE-2159-link-component.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+Improvement: "Refactor component to be more automated. Fixed known bugs. (Component: Link)"
```
Comment (grabkowski) [inline change_log/next/FE-2159-link-component.yml]: this change log entry does not correspond to changes made in this PR. "Fixed known bugs" doesn't say much.

File: src/components/create/create.style.js [hunk 2]
```diff
@@ -5,34 +5,38 @@ import createClassicStyle from './create-classic.style';
...
-  padding: 12px 12px 10px;
-  text-align: center;
-  font-weight: 700;
-  text-decoration: none;
-  width: 100%;
-
-  :hover {
-    background-color: ${({ theme }) => theme.colors.white};
+  
+  a{
```
Comment (grabkowski) [inline src/components/create/create.style.js]: please add a space before the brace

File: src/components/link/link.style.js [hunk 3]
```diff
@@ -5,48 +5,53 @@ import classicThemeForLinkAnchor from './link-classic.style';
...
-    `}
-    vertical-align: middle;
-    top: ${({ theme }) => (isClassic(theme) ? '-2px' : '')};
-  }
-  
-  &:hover {
-    cursor: pointer;
-    color: ${({ theme }) => theme.colors.secondary};
-  }
+  a{
```
Comment (grabkowski) [inline src/components/link/link.style.js]: another missing space

File: change_log/next/FE-2159-link-component.yml (hunk 1 above)
Comment (dawidzarzycki) [reply change_log/next/FE-2159-link-component.yml]: You are right. Updated

File: src/components/create/create.style.js (hunk 2 above)
Comment (dawidzarzycki) [reply src/components/create/create.style.js]: Updated

File: src/components/link/link.style.js (hunk 3 above)
Comment (dawidzarzycki) [reply src/components/link/link.style.js]: Updated

**Pull request merged**